syntax = "proto3";

package xray.app.proxyman.command;
option go_package = "github.com/xtls/xray-core/app/proxyman/command";

import "common/protocol/user.proto";
import "common/serial/typed_message.proto";

message AddUserOperation {
  xray.common.protocol.User user = 1;
}

message RemoveUserOperation {
  string email = 1;
}

message AlterInboundRequest {
  string tag = 1;
  xray.common.serial.TypedMessage operation = 2;
}

message AlterInboundResponse {}

service HandlerService {
  rpc AlterInbound(AlterInboundRequest) returns (AlterInboundResponse) {}
}
